app/views/application/_button.html.erb

```diff
@@ -0,0 +1,2 @@
+<% content = local_assigns.fetch(:content) %>
+<button class="btn rtsl-button border-black fg-black rounded-pill"><%= content %></button>
```

Comment:
i _think_ you can say `content` here without the `local_assigns.fetch`. was that not the case?

Reply:
i'd also consider using `button_tag` instead of this partial here. 

https://apidock.com/rails/ActionView/Helpers/FormTagHelper/button_tag